x-pack/plugin/stack/qa/rest/build.gradle

```diff
@@ -17,6 +18,11 @@ tasks.named("yamlRestTest").configure {
   systemProperty 'tests.rest.cluster.password', clusterCredentials.password
 }
 
+tasks.named("yamlRestCompatTest").configure {
```

Comment:
Same as above, let's try and consolidate this logic.

Reply:
6cd87e1